pkg/resourcecollector/resourcecollector.go

```diff
@@ -273,6 +281,22 @@ func skipOwnerRefCheck(annotations map[string]string) bool {
 	return false
 }
 
+// excludeControlledResources will check whther the given resource is part of
+// excludeControlledResourcesList. If present, it will not be included in the result
+// of the GetResources API.
+func (r *ResourceCollector) excludeControlledResources(name string, kind string) bool {
```

Comment:
Would be better to do this like other resource types in `objectToBeCollected()`. Create configmap.go in this package and add  a function called `configMapToBeCollected(...)`

Reply:
Sure. will fix it.

Reply:
Fixed the comment.